packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.10.3+3
 
+* CameraPlatfrom.createCameraWithSettings to allow recorded video fps and bitrate control.
```

Comment:
Nit: "Adds `CameraPlatfrom.createCameraWithSettings` to ..."

or just "Allows recorded video..."

Reply:
Changed